paasta_tools/tron_tools.py

```diff
@@ -98,7 +98,7 @@
 DEFAULT_AWS_REGION = "us-west-2"
 EXECUTOR_TYPE_TO_NAMESPACE = {
     "paasta": "tron",
-    "spark": "paasta-spark",
+    "spark": "tron",
```

Comment:
Why are we changing this? 
What impact would it have on existing infra?

Reply:
This is being changed because the Spark drivers will be run in the tron namespace. This would not have an impact on the existing infra because it is only being used in case of `executor: spark` being set in configs

Reply:
Why do we want to run the spark drivers on the tron namespace?